Jenkinsfile

```diff
@@ -158,6 +158,20 @@ pipeline {
             }
         }
 
+        stage('make dm-test') {
```

Comment:
Is this an experiment to figure out how long this stage will take?

Reply:
That is one reason, and the other is to run the docker with  `--privileged=true`  to run dm tests.

Reply:
@pohly do you have any clue why `ssh-keygen` is failing at this stage:

```
2019-11-08T12:10:18.773Z] + '[' '!' -e /go/src/github.com/intel/pmem-csi/_work/resources/id_rsa ']'

[2019-11-08T12:10:18.773Z] + ssh-keygen -N '' -f /go/src/github.com/intel/pmem-csi/_work/resources/id_rsa

[2019-11-08T12:10:18.773Z] + die 'failed to create /go/src/github.com/intel/pmem-csi/_work/resources/id_rsa'

[2019-11-08T12:10:18.773Z] + echo 'ERROR: failed to create /go/src/github.com/intel/pmem-csi/_work/resources/id_rsa'

[2019-11-08T12:10:18.773Z] ERROR: failed to create /go/src/github.com/intel/pmem-csi/_work/resources/id_rsa

[2019-11-08T12:10:18.773Z] + exit 1
```

Reply:
Does `_work/resources` exist?

Reply:
It should be created by the previous command line, which does not throw any error. So I assume it exists.

Reply:
When printing `ssh-keygen` output revealed that `ssh-keygen` command not found. https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/blue/organizations/jenkins/pmem-csi/detail/PR-420/20/pipeline#step-63-log-114

Reply:
> Is this an experiment to figure out how long this stage will take?

@pohly  Running device manager tests in VM took around 3 and half minutes in CI.
 **make dm-test - 3m 23s**